README.md

```diff
@@ -30,17 +30,20 @@ Requires a running [docker](https://docker.com) engine.
 
 ```
 docker run \
-  -e LIQUIDATOR_ACCOUNT_OWNER=0x2c7536E3605D9C16a7a3D7b1898e529396a65c23 \
-  -e LIQUIDATOR_ACCOUNT_OWNER_PRIVATE_KEY=0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318 \
-  -e ETHEREUM_NODE_URL=mainnet.infura.io/v3/YOUR_INFURA_KEY \
+  -e WALLET_ADDRESS=0x2c7536E3605D9C16a7a3D7b1898e529396a65c23 \
+  -e WALLET_PRIVATE_KEY=0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318 \
+  -e ETHEREUM_NODE_URL=https://mainnet.infura.io/v3/YOUR_INFURA_KEY \
+  -e SOLO_LIQUIDATIONS_ENABLED=true \
+  -e PERP_LIQUIDATIONS_ENABLED=true \
+  -e SOLO_EXPIRATIONS_ENABLED=false \
   dydxprotocol/liquidator
 ```
 
 ## Overview
 
 This service will automatically liquidate undercollateralized and expired Accounts on the [dYdX Solo Protocol](https://github.com/dydxprotocol/solo). Liquidations on dYdX happen internally between Accounts, so no actual ERC20 token movements occur.
```

Comment:
Update this overview to make it make sense for the perpetual as well. Make it clear it can be used for either or both together